### PR TITLE
python -m raiden working without pip install

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: raiden/ui/cli.py [hunk 1]
```diff
@@ -1012,7 +1012,10 @@ def _run_app(self):
             tasks.append(console)
 
         # spawn a greenlet to handle the version checking
-        tasks.append(gevent.spawn(check_version))
+        version = get_system_spec()['raiden']
+        if version is not None:
+            tasks.append(gevent.spawn(check_version))
```
Comment (andrevmatos) [inline raiden/ui/cli.py]: `check_version` needs `current_version`

File: requirements.txt [hunk 2]
```diff
@@ -23,6 +23,7 @@ eth-utils==1.2.0
 structlog
 colorama
 py-solc
+pytest
```
Comment (andrevmatos) [inline requirements.txt]: Why `pytest` in `requirements.txt`?

Comment (hackaugusto) [reply requirements.txt]: smoketest uses it :(

File: raiden/ui/cli.py (hunk 1 above)
Comment (hackaugusto) [reply raiden/ui/cli.py]: uhhh, thanks

File: requirements.txt (hunk 2 above)
Comment (andrevmatos) [reply requirements.txt]: Ok, need to remove it from `requirements-dev.txt` then, it breaks being in both

Comment (hackaugusto) [reply requirements.txt]: what breaks?

Comment (andrevmatos) [reply requirements.txt]: https://travis-ci.org/raiden-network/raiden/jobs/428128989#L579

Comment (hackaugusto) [reply requirements.txt]: ohh, gracias

File: constraints.txt [hunk 3]
```diff
@@ -0,0 +1,92 @@
+aniso8601==3.0.2
```
Comment (andrevmatos) [inline constraints.txt]: How about naming it something like `requirements-lock.txt`, to comply with other similar systems?

Comment (hackaugusto) [reply constraints.txt]: wait, this file was not suppose to be in this PR. Pleas move the discussion to : https://github.com/raiden-network/raiden/pull/2469

**Pull request merged**